origin.spec

```diff
@@ -35,6 +35,9 @@
 %global make_redistributable 1
 %endif
 
+# by default build the test binaries for Origin
+%global build_tests 1
+
```

Comment:
What the line above means
`%{!?build_tests: %global build_tests 1 }`
is "If build_tests is not defined, then set build_tests to 1, else use the defined build_tests variables"
That is the standard rpmbuild way of making a variable changeable via the command line.
If you want to stick with your way, then that is fine, I will give it a thumbs up.